app/retail/templates/home/index2020.html

```diff
@@ -129,7 +140,7 @@ <h2>Here to earn?</h2>
           Ethereum and other tokens. Find mentorship by connecting with
           talented developers and leaders of Ethereum’s top companies.
         </p>
-        <a class="btn fe-pulse" href="/hackathon-list"><span>Join a Hackathon</span></a>
+        <a class="btn fe-pulse" onclick="loadOnboardingModal()"><span>Join a Hackathon</span></a>
```

Comment:
Not sure what this index2020 page is but if you already dismissed the modal then this button will not work anymore, the modal should open on hackathon list not on top of the landing, and this button should always go to hackathon list for SEO reasons.

Reply:
Agree with @octavioamu

Reply:
Oops